### PR TITLE
Single member profile view

### PR DESCRIPTION
**Pull request by latinadeveloper**
## Overview

- Add member profile page.
- Put social links in a component.
- Fixed path so icons show icon on head(title) across all pages. 
- Make all card clickable so it goes to the member page instead of just their name. 
- Move Individual contributor to be part of the clickable area to go to the profile page.

## Relevant Issue

Link issue this PR is related to [Issue 19](https://github.com/Latina-Dev/latina-dev/issues/19)

## ✅️ By submitting this PR, I have verified the following

- [x] I added a description
- [x] I linked the relevant issue(s)
- [ ] If adding new copy, I made sure the content is clear for people to read and a tool like [Grammarly](https://grammarly.com/) was utilized to check for spelling and grammar

### CODE REVIEW

**Comment by latinadeveloper** [comment]
@EllaCodes4

File: types/members.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ type MemberLevel = "Student" | "Individual Contributor" | "Leader";
 
 export interface MemberInterface {
   name: string; // full name, e.g. Frances Coronel
+  affiliation: string // company or school, eg. Google
```
Comment (FrancesCoronel) [inline types/members.ts:5]: technically it also includes the title, right?

**Comment by FrancesCoronel** [comment]
I took a peek and left 3 comments - everything looks very good!

File: types/members.ts (hunk 1 above)
Comment (FrancesCoronel) [reply types/members.ts:5]: sounds good, i was just thinking it might be worth updating the comment for that too

Comment (FrancesCoronel) [reply types/members.ts:5]: also it should be optional since we are not requiring it

Comment (FrancesCoronel) [reply types/members.ts]: made some updates!

**Review by FrancesCoronel — APPROVED** [review]
Made some minor updates related to feedback (feel free to check the commit history) 

This looks ready to merge!

**Comment by FrancesCoronel** [comment]
Looks like Prettier recommends single quotes too so I'll update those in the settings because I've been using double quotes (whoops)

**Pull request merged**